CONTRIBUTING.md

```
@@ -87,21 +87,28 @@ poetry run poe test_unit
 
 #### Integration Tests
 
-To run integration tests, you'll need a standalone rippled node running with WS port `6006` and JSON RPC port `5005`. You can run a docker container for this:
+To run integration tests, you'll need a standalone `xrpld` node running with WS port `6006` and JSON RPC port `5005`. You can run a docker container for this:
 
 ```bash
```

markdownlint-cli2 (0.22.1)

[warning] Code block style
Expected: indented; Actual: fenced

(MD046, code-block-style)